templates/components/drop-card.html

```diff
@@ -1,3 +1,11 @@
+<script>
+    function getLocalDate(dateString) {
+        var months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'June', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
```

Comment:
Note: this won't get localized by translators since those string won't get extracted. Probably not a huge deal for now since the NFT site itself is not localized.